docs/contracts/inheritance.rst

```diff
@@ -244,6 +244,27 @@ overridden when used with multiple inheritance:
 
   Functions with the ``private`` visibility cannot be ``virtual``.
 
+Public state variables can override external functions with no parameters if the
```

Comment:
What about
```
interface ERC20 {
  function balanceOf(address) external view returns (uint);
}
contract C {
  mapping(address => uint) public balanceOf;
}
```

Reply:
That compiles.. should it?

Reply:
ah, I didn't add the `is IRC20` part in my test

Reply:
still works with the fixes

Reply:
Then please change the documentation.